ppa/archive/solr.py

```diff
@@ -126,6 +126,7 @@ class ArchiveSearchQuerySet(SolrQuerySet):
         "collections",
         "source_t",
         "image_id_s",
+        "first_page_i",
```

Comment:
does `group_id` not get added as well?

Reply:
I don't think we need it in the returned result, only server side for grouping works and pages